pkg/oc/bootstrap/docker/up.go

```diff
@@ -628,14 +628,6 @@ func GetDockerClient() (dockerhelper.Interface, error) {
 			glog.Infof("DOCKER_CERT_PATH=%s", dockerCertPath)
 		}
 	}
-	// FIXME: Workaround for docker engine API client on OS X - sets the default to
-	// the wrong DOCKER_HOST string
-	if runtime.GOOS == "darwin" {
```

Comment:
@deads2k @csrwng seems like not needed anymore... this was confusing detection of "real" remote docker....